src/Exception/ApiException.php

```diff
@@ -32,12 +36,6 @@ public function __construct($response)
 
             $this->message = $message;
         }
-
-        if (!empty($response->getHeader('Request-ID')[0])) {
```

Comment:
This change is intentional, the getRequestId() function pull directly from the response so storing it as part of the exception isn't needed anymore. Was supposed to remove it before my last PR